src/pages/LoadUnload/components/DetailFeature/RoadSectionLoadUnload.tsx

```diff
@@ -60,70 +72,92 @@ export const LoadUnloadDetailFeatureRoadSectionLoadUnload = ({
     return Object.entries(grouped)
   }
 
+  const parkingSigns = useMemo(() => {
+    return roadSectionLoadUnload.properties.load_unload
+      .reduce((acc: ParkingSign[], cur: LoadUnloadRegime): ParkingSign[] => {
+        if (acc.includes(cur.additional_info as ParkingSign)) return acc
+        if (cur.additional_info === 'E01' || cur.additional_info === 'E02') {
```

Comment:
Do you want to be this specific or should we check if cur.additional_info is in SignCodeToImg?

Reply:
No, because as per request we do not want to show all the possible traffic signs that may exist in "additional info". For now, we are specifically interested in E01 and E02.